### ISSUE TITLE
create VM with multiple cdRoms

### ISSUE THREAD

**Issue by tenderitaf** [ansible/ansible-modules-core#3205]
##### Issue Type:

- Bug Report
- Feature Idea
##### Component Name:

vsphere_guest.py/vsphere_guest
##### Ansible Version:

ansible 1.9.1 
##### Summary:

The objectif is to try to create a VM with at least two CDROMs
##### Steps To Reproduce:

This a code snipet of my VM config 

```
vm_hardware:
  memory_mb: "{{ resource_mem_vm }}"
  num_cpus: "{{ resource_cpu_vm }}"
  osid: "{{ vsphere_guestosid }}"
  scsi: "{{ vsphere_guestscsi }}"
  vm_cdrom:
    cdrom1:
    type: "iso"
    iso_path: "/{{ vsphere_file_root_path }}/{{ clusterid }}/cd1.iso"
    cdrom2:
    type: "iso"
    iso_path: "/{{ vsphere_file_root_path }}/{{ clusterid }}/cd2.iso"
```

##### Expected Results:

have a VM with two CDRoms

**Comment by ansibot** [comment]
@chrrrles, @rhoop, ping. This issue is waiting on your response.
[click here for bot help](https://github.com/jctanner/ansibullbot/blob/ISSUE_TRIAGE_CLASS/ISSUE_HELP.md)

**Comment by bbgrk** [comment]
This is exactly what I need now. It's unfortunate that this is not supported currently. Is there any chance that this will be implemented?

**Comment by Cougar** [comment]
I'm not sure if 507aa1b is good enough for pull request and merge to the devel but this is what I use right now. So far I have used it successfully for CoreOS automatic installation (config drive ISO mounted s vm_cdrom2). I haven't found any other solution how to install CoreOS by Ansible.

There are some things that I don't like in this patch:
- copy-paste of `vm_cdrom` code. should be refactored to one piece of code
- `vm_cdrom` under `vm_hardware` which is different logic than `vm_disk` have

I also made another try to mimic `vm_disk` type hierarchy but then it is complicated to make the change work. Maybe the requirement for names like `cdrom1` and `cdrom2` or similar would help, but using ambiguous names doesn't work here.

Anyway, I'm not sure that the current code supports all possible `vm_cdrom` changes (adding, removing, changing image) either. I haven't need that but might be necessary for some scenarios.

Another limitation is the nuber of CD/DVD drives. These can be added only under IDE controller and AFAIK the limitation is 4 here. Maybe it is possible to add more IDE controllers but I'm not sure.
